### ISSUE TITLE
Error upon sending files on s3 (server)

### ISSUE THREAD

**Issue by panchenkoprog** [inlife/nexrender#275]
Hi! 
I have two questions:
1) is there a way to clear server-server queue and if yes then how?
2) I can't upload files on S3 and I receive such error: http://prntscr.com/pwlps1
- it doesn't clear for me where can I send credentials
- why does it return such url: https://s3-eu-west-2.amazonaws.com/panchenkoprog/myfolder/encoded.mp4 . When I upload files on S3 they have different structure: https://panchenkoprog.s3.eu-west-2.amazonaws.com/shopify.jpg.
```
{
    "module": "@nexrender/action-upload",
     "input": "encoded.mp4",
     "provider": "s3",
     "params": {
        "region": "eu-west-2",
        "bucket": "panchenkoprog",
        "key": "myfolder/encoded.mp4",
        "acl": "public-read"
    }
}
```
As you can see from my example: (bucket: panchenkoprog) is in different place. Maybe it cause the problem?

**Comment by inlife** [comment]
Hello,

1. Right now only manually calling GET /jobs/, and then calling DELETE /jobs/:uid on each one you want to remove. Or as an alternative, stopping server, removing the /temp/nexrender/db.json, and starting it again.

2. Regarding credentials for AWS, make sure to define env variables for access: https://github.com/inlife/nexrender/tree/master/packages/nexrender-provider-s3#installation
And regarding the structure, I believe those are equal paths

**Comment by panchenkoprog** [comment]
Thanks for the help! It helped me a lot to advance in my project.
I have another question:
Where should the ENV file be located? I understand that it should be specified for NodeJS assembly? I have NodeJS on my local server globally, but I did not deploy the NodeJS assembly, but simply use the binary file nexrender-worker-win64.exe and therefore can’t understand where the ENV settings file should be located? I start the nexrender-worker-win64.exe file from the console manually. Please tell me where the ENV settings file,should be located next to the nexrender-worker-win64.exe file?
Thanks in advance!

**Comment by inlife** [comment]
It doesn't have to be an ENV file, it is just a set of environment variables
https://www.computerhope.com/issues/ch000549.htm

**Comment by panchenkoprog** [comment]
Thank you very much!
You helped me a lot.
1) I set the environment variables and it worked =)
2) Yes, you are really right, the links are the same. http://prntscr.com/px4qh3

When the link was displayed in the console, I was able to go through it and get my file, and also on the other link (in the admin panel) was the same file. This is not a problem for me, but I still want to understand - why do different links lead to the same place? The links differ in the location of the basket name. 

Thanks in advance!
